iOS/Features/RewindJourney/Sources/RewindJourneyView/RewindJourneyViewController.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  RewindJourneyViewController.swift
+//  RewindJourney
+//
+//  Created by 전민건 on 11/22/23.
+//
+
+import UIKit
+
+import MSDesignSystem
+
+public final class RewindJourneyViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    private enum Metrix {
+        
+        //progress bar
+        enum Progressbar {
+            static var height: CGFloat = 4.0
+            static var inset: CGFloat = 4.0
+            static var defaultIndex: Int = 0
+        }
+        
+        //stackView
+        enum StackView {
+            static var inset: CGFloat = 12.0
+        }
+        
+        //musicView
+        enum MusicView {
+            static var height: CGFloat = 69.0
+            static var inset: CGFloat = 12.0
+            static var bottomInset: CGFloat = 34.0
+        }
+        
+    }
+    
+    // MARK: - Properties
+
+    var stackView = UIStackView()
+    var presentImageView = UIImageView()
+    let musicView = MSMusicView()
+    var progressViews: [MSProgressView]?
+    var preHighlightenProgressView: MSProgressView?
+    let leftTouchView = UIButton()
+    let rightTouchView = UIButton()
+    
```

Comment:
```suggestion
    
    // MARK: - Properties
    
```

MARK 주석 용도가 조금 헷갈릴 수 있겠네요..
노션 문서 업데이트 해두겠습니다 😅

https://www.notion.so/musicspot/Swift-c80e89fdf4ee41209c7afaed3d759e82?pvs=4#2ca9bc36ba1640939eb82712e2db8b65